apps/extension/src/app/pages/home/home.tsx

```diff
@@ -1,9 +1,82 @@
-import { useFlags } from '@app/features/feature-flags';
+import { Route } from 'react-router';
 
-import { HomeV1 } from './home-v1';
-import { HomeV2 } from './home-v2';
+import { HomePageSelectors } from '@tests/selectors/home.selectors';
+import { Box, Stack } from 'leather-styles/jsx';
+
+import { RouteUrls } from '@shared/route-urls';
+
+import { formatCurrency } from '@app/common/currency-formatter';
+import { whenPageMode } from '@app/common/utils';
+import { ActivityList } from '@app/features/activity-list/activity-list';
+import { Collectibles } from '@app/features/collectibles/collectibles';
+import { FeedbackButton } from '@app/features/feedback-button/feedback-button';
+import { PromoBanner } from '@app/features/promo-banner/promo-banner';
+import { Assets } from '@app/pages/home/components/assets';
+import { homePageModalRoutes } from '@app/routes/app-routes';
+import { ModalBackgroundWrapper } from '@app/routes/components/modal-background-wrapper';
+import { AccountCard } from '@app/ui/components/account/account.card';
+
+import { AccountActions } from './components/account-actions';
+import { HomeTabs } from './components/home-tabs';
+import { useHomePageState } from './use-home-page-state';
+
+const emptyBalanceDisplay = '-.--';
 
 export function Home() {
-  const { extension_revamp } = useFlags();
-  return extension_revamp ? <HomeV2 /> : <HomeV1 />;
+  const {
+    balance,
+    isFetchingBnsName,
+    isPrivateMode,
+    name,
+    togglePrivateMode,
+    toggleSwitchAccount,
+  } = useHomePageState();
+
+  return (
+    <Stack
+      data-testid={HomePageSelectors.HomePageContainer}
+      px={{ base: 0, md: 'space.05' }}
+      py={{ base: 0, md: 'space.07' }}
+      gap={{ base: 0, md: 'space.06' }}
+      width="100%"
+      bg="ink.1"
+      borderRadius="lg"
+      animation="fadein"
+      animationDuration="500ms"
+    >
+      <Box px={{ base: 'space.05', md: 0 }} pb={{ base: 'space.05', md: 0 }}>
+        <AccountCard
+          name={name}
+          availableBalance={
```

Comment:
I'll refactor this to use the `Balance` component when working on centralising the balances UI